.swift-version

```diff
@@ -1 +1 @@
-4.0
+5.0
```

Comment:
It should be the latest 4.2 we support. 
Otherwise RxAlamofire won't support Swift 4.x at all

Reply:
Agreed.